src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -142,6 +143,28 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         });
     }
 
+    void TermControlAutomationPeer::NotifyNewOutput(const std::wstring_view newOutput)
+    {
+        auto dispatcher{ Dispatcher() };
+        if (!dispatcher)
+        {
+            return;
+        }
+
+        dispatcher
+            .RunAsync(Windows::UI::Core::CoreDispatcherPriority::Normal, [&]() {
+                try
+                {
+                    RaiseNotificationEvent(AutomationNotificationKind::ActionCompleted,
+                                           AutomationNotificationProcessing::All,

```

Comment:
Should probably also add a note here saying to use `All` to allow for keyboard event interruptions.